### ISSUE TITLE
Warning in the log file- suspicious....

### ISSUE THREAD

**Issue by mlmaskey** [ucd-cws/calvin#51]
@nickrsan and @msdogan ,
I noticed a warning message the log file of CALVIN, which @nickrsan  recently implemented in the `calvin.py`.  This message  (below) tells you that the model model did not reach an optimal solution but founds the solution without getting the required iteration. It created me some doubt about the result, and I did not see any significant difference while you added the facilities in the system as well.   
What do you think? Is there a way to optimize iterations inside the program? Or shall we just ignore it. 
```
2020-08-25 20:50:09,749 - calvin - INFO - HSD203.1997-12-31_C18.1997-12-31 UB raised by 0.00 (0.00%)
**2020-08-25 20:50:09,767 - calvin - INFO - Warning: Debug mode maximum iterations reached. Will still try to solve without debug mode.**
2020-08-25 20:50:09,990 - calvin - INFO - Creating Pyomo Model (debug=False)
```
If you think it can be solved, let's have a meeting and have discussed it.

Thank you...
[calvin.log](https://github.com/ucd-cws/calvin/files/5285798/calvin.log)

**Comment by nickrsan** [comment]
Hi Mahesh, one more thing in addition to our discussion in Teams on this - this behavior isn't new or changed recently - it has been there a long time - just the way the log outputs the notice has changed. So, it's doing the same thing it has done for a long time, just telling you about it a bit differently.

**Comment by mlmaskey** [comment]
Well, I am not pointing out that it is from you, @nickrsan. I mean to say if there is something in the proceeding part of the code. I am just sensitive about that message. I apologize if you think another way.

**Comment by nickrsan** [comment]
Not a problem - I just wanted to be clear that it *isn't* something that change any time recently, before or after the workshop. It has been a constant behavior. I wanted to make sure to point it out since it seems like you feel something has changed that is causing the software to behave incorrectly. If that's the case, I don't think CALVIN changed - instead, I think we should examine the matrix tool still - it still makes sense to talk with Mustafa about what's happening with this message, but I wanted to be clear it's doing the same thing it has done for a long time now.

**Comment by mlmaskey** [comment]
Yes, this is my concern, @nickrsan. I am not just worry about my work. I am talking in general. 
 @msdogan, Please give us some time to fix it.
Thank you

**Comment by msdogan** [comment]
@mlmaskey I believe it is just an numerical error. Upper bound is raised by 0.00 (0.00%), which is a very small number but still showing up. Maybe the way it is programmed is very sensitive. I suggest just ignore it. Thanks.
